XS✔ ◾ Mention GitHub Discussions (#8194)
SSWConsulting/SSW.Rules.Content: rules/reply-done-and-delete-the-email/rule.md

### Tip #2: Provide details in your "Done"

> In any reply, [include relevant information](/dones-do-you-include-useful-details-in-your-done-email), such as [URLs](/dones-do-your-dones-include-a-url), bookmark to a comment on GitHub Discussions, screenshots, and pieces of code/text that have been updated. This allows others to check what was done straight away.

Reviewer: ```suggestion
In any reply, [include relevant information](/dones-do-you-include-useful-details-in-your-done-email), such as [URLs](/dones-do-your-dones-include-a-url), direct link to a comment on GitHub Discussions, screenshots, or pieces of code/text that have been updated. This allows others to check what was done straight away.
```
